src/InstagramClone.Api/Entities/User.cs

```diff
@@ -10,6 +10,10 @@ public class User
         public string LastName { get; set; }
         public string Avatar { get; set; }
         public DateTime CreatedDate { get; set; }
+        public ICollection<Post> Posts { get; set; }
+        public ICollection<UserFollower> Users { get; set; }
```

Comment:
what's the purpose of this property?

Reply:
I used the public ICollection<Post> Posts { get; set; } to configure a one to many relationship on line 27 in my context class.